staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -408,6 +408,7 @@ func (c *Cacher) startCaching(stopChannel <-chan struct{}) {
 		successfulList = true
 		c.ready.set(true)
 		klog.V(1).Infof("cacher (%v): initialized", c.objectType.String())
+		watchCacheInitializations.WithLabelValues(c.objectType.String()).Add(1)
```

Comment:
nit: you could use `.Inc()` instead of `.Add(1)`

Reply:
done